app/deliver_grant_funding/helpers.py

```diff
@@ -0,0 +1,44 @@
+from typing import TYPE_CHECKING, Optional
+
+from flask import redirect, session, url_for
+from flask.typing import ResponseReturnValue
+
+from app.common.data import interfaces
+from app.common.data.interfaces.collections import create_submission, delete_collection_test_submissions_created_by_user
+from app.common.data.types import SubmissionModeEnum
+from app.common.helpers.collections import SubmissionHelper
+
+if TYPE_CHECKING:
+    from app.common.data.models import Collection, Form
+
+
+def start_testing_submission(collection: "Collection", form: Optional["Form"] = None) -> ResponseReturnValue:
```

Comment:
1 small test for this:
- pre-existing test submission that gets deleted/wiped
- redirects go to expected URLs for form-based vs task-based